pkgs/development/libraries/libyaml/default.nix

```diff
@@ -14,7 +14,7 @@ stdenv.mkDerivation {
     homepage = http://pyyaml.org/;
     description = "A YAML 1.1 parser and emitter written in C";
     license = licenses.mit;
-    platforms = platforms.unix;
+    platforms = platforms.all;
```

Comment:
the attribute may as well be dropped then

Reply:
Nah then hydra will only build on linux.

Reply:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/release-lib.nix#L125

Reply:
Even if that wasn't the case (I wish it weren't), I'd think the explicit `all` is good to show intent.